tools/windows/install-help/cal/customaction-tests/ReplaceYamlProperties_ProcessTests.cpp

```diff
@@ -81,3 +81,28 @@ TEST_F(ReplaceYamlPropertiesTests, When_Process_Url_Set_And_Apm_Enabled_Correctl
   enabled: "disabled"
 )");
 }
+
+TEST_F(ReplaceYamlPropertiesTests, When_Process_Discovery_Enabled_Correctly_Replace)
```

Comment:
Can you add a test that sets both `PROCESS_DD_URL` and `PROCESS_DISCOVERY_ENABLED`? I think they will interact fine, but just to make sure.

Reply:
Sounds good, will add this.